testing/test_pluginmanager.py

```diff
@@ -98,6 +98,33 @@ def pytest_configure(self):
         config.pluginmanager.register(A())
         assert len(values) == 2
 
+    @pytest.mark.skipif(
+        not sys.platform.startswith("win"),
+        reason="requires a case-insensitive file system",
+    )
+    def test_conftestpath_case_sensitivity(self, pytester: Pytester) -> None:
+        config = pytester.parseconfig()
+        pytester.makepyfile(**{"tests/conftest.py": ""})
+
+        conftest = pytester.path.joinpath("tests/conftest.py")
+        conftest_upper_case = pytester.path.joinpath("TESTS/conftest.py")
+
+        mod = config.pluginmanager._importconftest(
+            conftest, importmode="prepend", rootpath=pytester.path
+        )
+        assert config.pluginmanager._name2plugin[str(conftest)] is mod
+
+        mod_uppercase = config.pluginmanager._importconftest(
+            conftest_upper_case, importmode="prepend", rootpath=pytester.path
+        )
+        assert (
+            config.pluginmanager._name2plugin[str(conftest_upper_case)] is mod_uppercase
+        )
+
+        # No str(conftestpath) normalization so conftest should be imported
```

Comment:
I'll note that this is *not* the behavior we want -- we want them to be the same. But as we agreed I will add the normalization in a separate PR.